includes/wikia/tasks/Tasks/ImageReviewTask.class.php

```diff
@@ -111,7 +111,6 @@ public function delete( $pageList, $suppress = false ) {
 
 		$success = $articlesDeleted == count( $pageList );
 		if ( !$success ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS108)